### PR TITLE
NM committee Scraper

### PR DESCRIPTION
**Pull request by EhtishamSabir**
New spatula-based scraper written for New Mexico.

### CODE REVIEW

**Review by johnseekins — APPROVED** [review]

File: scrapers_next/nm/committees.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+        for chamber, elems in all_committees.items():
+            for item in elems:
+                name, com_url = item.text, item.get("href")
+                if not com_url:
+                    continue
+
+                if "subcommittee" in name.lower():
+                    classification = "subcommittee"
+                    parent = name.lower().replace("subcommittee", "").title()
```
Comment (chrisyamas) [inline scrapers_next/nm/committees.py:89]: Make the same change here that you're making on the [Wyoming scraper](https://github.com/openstates/openstates-scrapers/pull/4458#discussion_r1106015163). There are only two committees so far with subcommittee in their name, and neither of them have clear parent, so just use that same kind of code block as what I suggested there to make them both `classification="committee"` and then raise a custom exception if any others are encountered in the future.

**Review by chrisyamas — CHANGES_REQUESTED** [review]
One minor change, data looks good other than that!

File: scrapers_next/nm/committees.py (hunk 1 above)
Comment (EhtishamSabir) [reply scrapers_next/nm/committees.py:89]: done...

**Review by chrisyamas — APPROVED** [review]
I added some final reformatting and fixes to ensure the name cleaning function was working properly, but it's good to go now. Approving and merging!

**Pull request merged**